core/src/toga/app.py

```diff
@@ -516,13 +517,39 @@ def exit_full_screen(self):
             self._impl.exit_full_screen(self._full_screen_windows)
             self._full_screen_windows = None
 
+    @property
+    def cursor_position(self):
+        """Return the cursor position with respect to the whole app."""
+        return self._impl.get_cursor_position()
+
+    @cursor_position.setter
+    def cursor_position(self, value: Tuple[int, int]):
+        """Set the cursor position with respect to the whole app."""
+        if not isinstance(value, tuple) or len(value) != 2:
+            print("Error! Invalid cursor position value or type.")
+            return
+        self._impl.set_cursor_position(value)
+
+    @property
+    def cursor_visible(self):
+        """Return the status of cursor visibility for the whole app."""
+        return self._impl.is_cursor_visible()
+
+    @cursor_visible.setter
+    def cursor_visible(self, value: bool):
+        """Set the cursor visibility for the whole app."""
+        if not isinstance(value, bool):
+            print("Error! Invalid cursor visible value.")
+            return
+        self._impl.set_cursor_visible(value)
+
     def show_cursor(self):
-        """Show cursor."""
-        self._impl.show_cursor()
+        """Show cursor for the whole app."""
+        self._impl.show_cursor(self.windows)
```

Comment:
I'm not sure I see why "self.windows" is needed as an argument here. If an implementation needs the list of windows, they can get it from `app.windows`. 

Also - the fact that you're changing an existing implementation should be a red flag. You're calling an show_cursor()/hide_cursor() on the cocoa backend...which already has an implementation of show/hide cursor that *doesn't* take the windows arguments.